Source/UI/Miscellaneous/IPFHelpViewController.m

```diff
@@ -13,7 +12,7 @@ - (instancetype)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibB
   self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];
 
   if (self != nil) {
-    self.title = NSLocalizedString(@"Help", @"Help screen title");
+    self.title = NSLocalizedString(@"TestHelper.title", @"Help screen title");
```

Comment:
Won't that show a robotic message for locales where we don't have a strings file?

Reply:
No I tested it by switching region and language. Unless a localization file is provided it just uses the default english localization